src/main/resources/view/ExamListCard.fxml

```diff
@@ -28,7 +27,7 @@
         </Label>
         <Label fx:id="name" styleClass="cell_big_label" text="\$exam_name" />
       </HBox>
-         <Label text="Max Score: " styleClass="cell_small_label"/>
+         <Label styleClass="cell_small_label" text="Max Score: " />
```

Comment:
Hmm this is an interesting swap in order. I'm assuming Scene Builder did this

Reply:
That is indeed interesting. Looks like SceneBuilder is insightfully trying to parallel life, where it does not place maximum scores as its first priority, as we all should.